src/sas/qtgui/Calculators/GenericScatteringCalculator.py

```diff
@@ -562,7 +567,9 @@ def change_is_avg(self):
         # update the averaging option fromthe button on the GUI
         # required as the button may have been previously hidden with
         # any value, and preserves this - we must update the variable to match the GUI
-        self.is_avg = (self.cbOptionsCalc.currentIndex() == 1)
+        self.is_avg = (self.cbOptionsCalc.currentIndex() in (1,2))
```

Comment:
Since you only want to reject item 0, maybe this would look nicer?
`self.is_avg = self.cbOptionsCalc.currentIndex() != 0`

Reply:
I considered it, but decided against it in the case that we would add more options/indexes in the future- does it make any difference functionally, or is it just a cosmetic fix?